services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -907,22 +911,28 @@ export class WorkflowService {
         ),
       };
 
-      this.__validateWorkflowDefinitionContext(workflowDef, {
-        ...context,
-        documents: context?.documents?.map(
-          (document: DefaultContextSchema['documents'][number]) => ({
-            ...document,
-            decision: {
-              ...document?.decision,
-              status: document?.decision?.status === null ? undefined : document?.decision?.status,
-            },
-            type:
-              document?.type === 'unknown' && document?.decision?.status === 'approved'
-                ? undefined
-                : document?.type,
-          }),
-        ),
-      });
+      try {
+        this.__validateWorkflowDefinitionContext(workflowDef, {
+          ...context,
+          documents: context?.documents?.map(
+            (document: DefaultContextSchema['documents'][number]) => ({
+              ...document,
+              decision: {
+                ...document?.decision,
+                status:
+                  document?.decision?.status === null ? undefined : document?.decision?.status,
+              },
+              type:
+                document?.type === 'unknown' && document?.decision?.status === 'approved'
+                  ? undefined
+                  : document?.type,
+            }),
+          ),
+        });
+      } catch (error) {
+        console.log('VALIDATION ERRORTS', JSON.stringify(error));
```

Comment:
The error here should be logged out without this try/catch + console.log here. Should probably either remove this before merging or at least use our logger.

Reply:
It was leftover and should not be there